integration-test/src/main/java/com/scalar/db/api/DistributedStorageIntegrationTestBase.java

```diff
@@ -526,6 +526,37 @@ public void scan_ScanWithLimitGiven_ShouldReturnGivenNumberOfResults()
         .isEqualTo(Optional.of(new IntValue(COL_NAME4, 2)));
   }
 
+  @Test
+  public void
+      scan_ScanWithClusteringKeyRangeAndConjunctionsGiven_ShouldRetrieveResultsOfBothConditions()
```

Comment:
I only added very basic tests for `Scan`/` ScanWithIndex` with the where clause since various patterns of conditions are being tested with cross-partition scan filtering.